README.md

```diff
@@ -768,14 +797,22 @@ This is useful for when you need to do a subsequent computation using the inner
 **Signature:**
 
 ```typescript
-type AndThenFunc = (t:  T) => ResultAsync<U, E> | Result<U, E>
-andThen<U>(f: AndThenFunc): ResultAsync<U, E> { ... }
+class ResultAsync<T, E> {
+  andThen<U>(
+    callback: (value: T) => Result<U, E> | ResultAsync<U, E>
+  ): ResultAsync<U, E> { ... }
+}
+```
+
+```typescript
+// Note that the latest version (v4.1.0-beta) lets you return distinct errors as well.
+// If the error types (E and F) are the same (like `string | string`)
+// then they will be merged into one type (`string`)
 
 class ResultAsync<T, E> {
-  // Note that the latest version (v4.1.0-beta) lets you return distinct errors as well.
-  // If the error types (E and F) are the same (like `string | string`)
-  // then they will be merged into one type (`string`)
-  andThen<U, F>(f: (t: T) => Result<U, F> | ResultAsync<U, F>): ResultAsync<U, E | F> { ... }
+  andThen<U, F>(
+    callback: (value: T) => Result<U, F> | ResultAsync<U, F>
+  ): ResultAsync<U, E | F> { ... }
```

Comment:
Why is there a second `andThen` signature?

Reply:
idk, I saw you had two listed already one for before v4.1.0-beta and one for after. Thought maybe you wanted to keep both. If not I can remove the old one